### PR TITLE
NoopDriver now uses ShaderModel::GL_ES_30 on WASM

### PR DESCRIPTION
**Pull request by lincolnfrog**
This allows us to do WASM-based unit tests using the NOOP driver.

### CODE REVIEW

File: filament/backend/src/noop/NoopDriver.cpp [hunk 1]
```diff
@@ -31,7 +31,7 @@ NoopDriver::NoopDriver() noexcept : DriverBase(new ConcreteDispatcher<NoopDriver
 NoopDriver::~NoopDriver() noexcept = default;
 
 backend::ShaderModel NoopDriver::getShaderModel() const noexcept {
-#if defined(ANDROID) || defined(IOS)
+#if defined(ANDROID) || defined(IOS) || defined(EMSCRIPTEN)
```
Comment (romainguy) [inline filament/backend/src/noop/NoopDriver.cpp:34]: Everywhere else we use:
`#if defined(__EMSCRIPTEN__)`

@prideout can you confirm?

Comment (prideout) [reply filament/backend/src/noop/NoopDriver.cpp:34]: Yes good catch Romain, let's be consistent with the rest of Filament.

Comment (lincolnfrog) [reply filament/backend/src/noop/NoopDriver.cpp:34]: that's how I had it originally but I figured we should match ANDROID and IOS. It does work FWIW.

Comment (prideout) [reply filament/backend/src/noop/NoopDriver.cpp:34]: The underscored version is more correct as it allows for emscripten's "strict" mode to be enabled, and is consistent with the rest of Filament.

Comment (lincolnfrog) [reply filament/backend/src/noop/NoopDriver.cpp]: Done.

**Pull request merged**